docs/summary.md

```diff
@@ -1,5 +1,18 @@
 # Table of contents
 
-* [Get Started](start.md)
-* [Credentials](credentials.md)
+* [AnyCloud Documentation](about.md)
+* [Tutorial](tutorial.md)
+* [How-to guides](how-to/README.md)
+  * [Define a custom app ID](how-to/custom-app.md)
+  * [Deploy to GCP](how-to/gcp.md)
+  * [Deploy to Azure](how-to/azure.md)
+  * [Pass environment variables](how-to/env.md)
+  * [Deploy an app to multiple clouds and/or regions](how-to/multi.md)
```

Comment:
+1 on this, still missing the `Deploy to AWS` section. It's mostly copy-paste from the tutorial, but it should be explicitly called out separately for someone looking for it that doesn't want to go through "the whole tutorial"

Reply:
> Would reorder this to have a sense of structure like options, deploys, multicloud:
> 
> * [Define a custom app ID](how-to/custom-app.md)
> * [Pass environment variables](how-to/env.md)
> * [Deploy to GCP](how-to/gcp.md)
> * [Deploy to Azure](how-to/azure.md)
> * [Deploy an app to multiple clouds and/or regions](how-to/multi.md)
> 
> Understand that AWS still in progress, right?

I would put environment variables on top of GCP, but not Azure.

Reply:
> +1 on this, still missing the `Deploy to AWS` section. It's mostly copy-paste from the tutorial, but it should be explicitly called out separately for someone looking for it that doesn't want to go through "the whole tutorial"

`Deploy to AWS` is part of the tutorial and called out explicitly at the beginning of the docs so it seems unnecessary to copy paste or call it out explicitly again?

Reply:
Blocking on this one.

Reply:
I already addressed this here https://github.com/alantech/anycloud/pull/34/files#r600153220. I couldn't even find this message thread again afterwards. To be clear this absolutely seems like and not something that should be blocking and I already expressed that it is wasteful and overwhelming to go through the amount of comments on this PR and I would address in follow up PR.

Reply:
That's not what matters. End users will come into these docs not knowing the structure. They will have a question they want to get and answer for and they will do their own searching through the docs for it.

If their question is: "How do I use this with AWS?" and they go to the "How to deploy..." section and don't see it listed,they could justifiably come to the conclusion that we don't support AWS.

So this is a missing piece that I consider a blocking concern and I cannot fathom why you don't see it the same way.

Reply:
I already addressed that before you made your blocking comment/review in this commit https://github.com/alantech/anycloud/pull/34/commits/0e1eebee1c8e1fec64ea16a16c0de5235ff6e8e8
as I mentioned before but I pasted the wrong link. As for what matters, imo is not something that I believe should be blocking as I mentioned in discord https://discord.com/channels/682284785519165454/793276115191332904/824732609103659038. I asked if I could address in follow up PRs since I can't really keep track of everything that is going on here and would address it separately in a follow up PR. This is something we have done often in the past so not sure why we are diverging here, but again I already addressed it. The fact that you missed my commit kinda proves the difficulty I am also having with Github and this PR in particular

Reply:
That solution wasn't missed. It's just below my bar? This documentation is *the way* people will be learning about `anycloud` and how to use it, so I'm sorry if you think I'm being picky, but I believe we need to make good first impressions here.

I was going to suggest including an animated gif of using the `anycloud` CLI on the main README, but I decided to be quiet about that for the first real version of the docs.

Reply:
I am happy to address your comment and make the documentation better I just thought you hadn't seen that. Per my question, I want to address it this PR is getting nightmarish to track so I am asking if I address that in a follow up PR?